sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs

```diff
@@ -76,21 +76,21 @@ public async Task RunSamplesAsync(DigitalTwinsClient client)
 
             string basicDtPayload = JsonSerializer.Serialize(basicTwin);
 
-            await client.CreateDigitalTwinAsync(basicDtId, basicDtPayload);
+            await client.CreateDigitalTwinAsync<object>(basicDtId, basicDtPayload);
```

Comment:
We should change this to receive the response (for the etag) and use an actual type instead of object.